python/mlc_chat/model/phi/phi_model.py

```diff
@@ -157,8 +185,16 @@ def forward(self, hidden_states: Tensor, attention_mask: Tensor, total_seq_len:
             total_seq_len,
         )
 
+        if self.tensor_parallel_shards > 1:
+            attn_outputs = op.ccl_allreduce(attn_outputs, "sum") - self.mixer.out_proj.bias
```

Comment:
@jinhongyii we will have to consider the case where `bias` needs to be fused before allreduce